objects/rct1/ride/rct1.ride.fruity_ices_stall/object.json

```diff
@@ -0,0 +1,77 @@
+{
+    "id": "rct1.ride.fruity_ices_stall",
+    "authors": ["Chris Sawyer", "Simon Foster"],
+    "version": "1.0",
+    "sourceGame": "rct1",
+    "isCompatibilityObject": true,
+    "objectType": "ride",
+    "properties": {
+        "type": "food_stall",
+        "category": "stall",
+        "clearance": 56,
+        "sells": "ice_cream",
+        "carsPerFlatRide": 1,
+        "carColours": [
+            [
+                ["light_blue", "black", "black"]
+            ]
+        ]
+    },
+    "images": [
+        { "path": "images/preview.png", "format": "raw" },
+        "",
+        "",
+        "$CSG[60625..60628]"
+    ],
+    "noCsgImages": [
+        { "path": "images/preview.png", "format": "raw" },
+        "",
+        "",
+        { "path": "images/nocsg_0.png", "x": -23, "y": -44 },
+        { "path": "images/nocsg_1.png", "x": -23, "y": -44 },
+        { "path": "images/nocsg_2.png", "x": -28, "y": -46 },
+        { "path": "images/nocsg_3.png", "x": -28, "y": -39 }
+    ],
+    "strings": {
+        "name": {
+            "en-GB": "Fruity Ices Stall",
```

Comment:
RCT1 referred to this as the "Ice cream stall", RCT2 as "Fruity Ices Stall", possibly in an attempt to differentiate it from the Ice Cream Cone Stall.

Should we go with RCT1 description, or keep RCT2’s? What do you reckon?

Reply:
Imo i'd just keep the name and descriptions the same as RCT2 where applicable. RCT1 has a lot of weird naming differences that'd just be confusing if you were to place them in an RCT2 context (eg: The lay-down coaster is known as the flying coaster in RCT1) especially since the RCT2 ride type names and descriptions are still used. Plus it would probably be easier for translation purposes since the same translations can be used for both RCT1 & RCT2 objects in most cases while if different strings were used those would have to be separately translated.